examples/ads.amp.html

```diff
@@ -1013,6 +1014,18 @@ <h2 id="purch">Purch 300x250</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="relap">Relap</h2>
+  <amp-ad width=auto height=750
```

Comment:
`relap` is about content recommendation with ads. The amount of ads depends on the publisher. The widget can be without any ads, have some or consist of only ads.